ephypype/fif2ts.py

```diff
@@ -19,3 +19,66 @@ def ep2ts(fif_file):
     save_path = op.abspath('ts_epochs.npy')
     save(save_path, data)
     return save_path
+
+
+def create_ts(raw_fname):
```

Comment:
what is `ts`? time series?

Reply:
and also should this be a private function? It's not used in the example ...

Reply:
> and also should this be a private function? It's not used in the example ...

yes, it's used by Fif2Ts class 

> and also should this be a private function? It's not used in the example ...

yes, now it's only for internal use, thus is it better to make it private?

Reply:
`ts` is confusing for me. Why not call it `get_raw_array` or `save_raw_array` or something like that? The name should tell me in one glance what the function is supposed to do. Using abbreviations does not help ...

Reply:
> `ts` is confusing for me. Why not call it `get_raw_array` or `save_raw_array` or something like that? The name should tell me in one glance what the function is supposed to do. Using abbreviations does not help ...

done! I choose _get_raw_array